exotica_python/src/Exotica.cpp

```diff
@@ -476,16 +496,6 @@ PYBIND11_MODULE(_pyexotica, module)
     py::class_<KinematicTree, std::shared_ptr<KinematicTree>> kinematicTree(kin, "KinematicTree");
     kinematicTree.def("publishFrames", &KinematicTree::publishFrames);
 
-    py::class_<KinematicFrameRequest, std::shared_ptr<KinematicFrameRequest>> kinFrameReq(kin, "KinematicFrameRequest");
```

Comment:
These are classes only used internally. I would like to keep the interface clear. 
It has very limited use for debugging. If there is a problem with a frame request, it should have been dealt with in c++. You don't even get to construct those classes when you are defining new task maps.